plugin/backend_checkouts_test.go

```diff
@@ -27,6 +28,144 @@ func TestCheckOuts(t *testing.T) {
 	t.Run("write set", WriteReserve)
 	t.Run("add service account", AddAnotherServiceAccount)
 	t.Run("remove service account", RemoveServiceAccount)
+
+	t.Run("check initial status", CheckInitialStatus)
+	t.Run("check out account", PerformCheckOut)
+	t.Run("check updated status", CheckUpdatedStatus)
+	t.Run("normal check in", NormalCheckIn)
+	t.Run("return to initial status", CheckInitialStatus)
+	t.Run("check out again", PerformCheckOut)
+	t.Run("check updated status", CheckUpdatedStatus)
+	t.Run("force check in", ForceCheckIn)
+	t.Run("check all are available", CheckInitialStatus)
```

Comment:
For now they're sunny path tests. We're going to need to leave negative tests out of scope for this iteration because we need to get this all merged by the end of the week. We do have additional testing built into our timeline, so I can definitely add negative tests once we have this in the beta.

Reply:
Marking this as resolved because I don't think there's anything else to do here in this PR.